rep-0003.rst

```diff
@@ -233,6 +233,66 @@ Buildsystem support:
 
   - build from source
 
+Lunar Loggerhead (May 2017 - May 2019)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 25
+
+Architecture Supported:
+
+- amd64
+- arm32
+- arm64
+
+Minimum Requirements:
+
+- C++11
+
+  - GCC 4.9 on Linux, as it's the version that Debian Jessie ships with
+
+- Python 2.7
+
+  - Python 3.4 not required, but testing against it is recommended
```

Comment:
That's a good point, all supported platforms will ship with python3.5.x. The reason was the same as https://github.com/ros-infrastructure/rep/pull/123#discussion_r95849731 to stay consistent with Kinetic and avoid people from branching out for Lunar. I'll change it to 3.5 here. Thanks for pointing it out